### PR TITLE
docs: add decision record about the release process

### PR DESCRIPTION
**Pull request by paullatzelsperger**
## Description
Adds a decision-record about the release process

## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [x] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

**Comment by paullatzelsperger** [comment]
\cc @matgnt @borisrizov-zf @dr-markus

**Review by wolf4ood — APPROVED** [review]

File: docs/decision-records/2023-09-22-release-process/README.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Defining a simple release process
+
+## Decision
+
+The release process for the specification should be simple and easy to understand. We will not follow GitFlow, but
```
Comment (jimmarino) [inline docs/decision-records/2023-09-22-release-process/README.md]:
```suggestion
The release process for the specifications should be simple and easy to understand. We will not follow GitFlow, but
```

File: docs/decision-records/2023-09-22-release-process/README.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+## Rationale
+
+With code artefacts it is oftentimes required to be able to back-port bugfixes or even features, or maintain older
+versions. Here, the deliverable is merely a compressed archive with static non-code content, so the release process can
+be kept fairly simple and linear.
+
+## Approach
+
+At a minimum, every release should be tagged on the `main` branch using semantic versioning, starting at `0.1.0`. Then,
```
Comment (jimmarino) [inline docs/decision-records/2023-09-22-release-process/README.md]:
```suggestion
Every release will be tagged on the `main` branch using semantic versioning, starting at `0.1.0`. Then,
```

File: docs/decision-records/2023-09-22-release-process/README.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+using that tag, a GitHub Release is to be created, for example using the `ncipollo/release-action` GitHub action.
+Finally, a compressed archive containing the current specification documents should be uploaded as artefact to the
```
Comment (jimmarino) [inline docs/decision-records/2023-09-22-release-process/README.md]:
```suggestion
Finally, a compressed archive containing the current specification documents will be uploaded as artefact to the
```

**Review by jimmarino — APPROVED** [review]
Few minor wording changes

**Pull request merged**